search-app/docs/api.md

```diff
@@ -1694,64 +1694,64 @@ __Sample response__
 
 ### <a name="granule-search-by-parameters"></a> Granule Search By Parameters
 
```

Comment:
We should add a short paragraph here to explain search performance for granule searches is significantly improved by including an identifier that limits the search to a certain collection or subset of collections. Examples of parameters which limit the scope of the search include collection_concept_id, short_name, entry_title, or provider.

Reply:
Thanks, Done.